controller_interface/include/controller_interface/multi_interface_controller.h

```diff
@@ -172,25 +172,10 @@ std::string enumerateElements(const T& val,
  * };
  * \endcode
  *
- * \tparam T1 Hardware interface type.
+ * \tparam T... Hardware interface types.
  * This parameter is \e required.
- *
- * \tparam T2 Hardware interface type.
- * This parameter is \e optional. Leave unspecified if controller only claims
- * resources from a \e single hardware interface.
- *
- * \tparam T3 Hardware interface type.
- * This parameter is \e optional. Leave unspecified if controller only claims
- * resources from \e two hardware interfaces.
- *
- * \tparam T4 Hardware interface type.
- * This parameter is \e optional. Leave unspecified if controller only claims
- * resources from \e three hardware interfaces.
- *
- * \pre When specified, template parameters \c T1 to \c T4 must be different
```

Comment:
This could potentially also be validated using `static_assert`; it wasn't checked previously.